app/posts/get-an-identity/2023-06-07-adding-preferred-name.md

```diff
@@ -0,0 +1,47 @@
+---
+  title: Collecting a preferred name in addition to official name 
+  date: 2023-06-07
+  related:
+    sections:
+      - title: Links
+        items:
+          - text: Avoiding duplicate names
+            href: /get-an-identity/avoiding-duplicate-names/
+---
+
+With our recent work on names, we’ve tried to reduce duplication of name and date of birth. Teaching ID will hold first, middle, and last names. If the teaching ID account is linked to DQT (Database of Qualified Teachers), we’ll defer to those names, so we’ll just have one set of names. The DQT names require documentation to change – as we need to make sure teaching certificates are issued in the correct name. For now, we’re calling that the ‘official name’.
+
+In addition to the official name, we’re also going to add what we call ‘preferred name’. This will be a name that users can provide to tell us how they’d preferred to be called. Unlike the official name, they will not need to give any documentation for it – they can provide whatever they like.
+
+We think this will be helpful where:
+
+* The full official name is longer or is not how the user usually refers to themselves
+* The user prefers a shortened form of their first name (Jess for Jessica, etc)
+* Services that do not need a full official name can defer to what the teacher has selected
+
```

Comment:
The first letter after a bullet should be lowercase where there's a lead in line.

Reply:
Oops! I'm going to blame word autocorrect for that one.